Nutraceuticals - Response to Reviewers (#877)
greenelab/covid19-review: content/21.nutraceuticals.md

It has been shown that as little as 0.1 g/d of vitamin C can maintain normal plasma levels of vitamin C in healthy individuals, but higher doses of at least 1-3 g/d are required for critically ill patients in ICUs [@doi:10.3390/nu11040708].
Indeed, vitamin C deficiency appears to be common among COVID-19 patients [@doi:10.1016/j.medidd.2020.100064; @doi:10.1186/s13054-020-03249-y].
COVID-19 is also associated with the formation of microthrombi and coagulopathy [@doi:10.1172/JCI140183] that contribute to its characteristic lung pathology [@doi:10.1183/16000617.0240-2020], but these symptoms can be ameliorated by early infusions of vitamin C to inhibit endothelial surface P-selectin expression and platelet-endothelial adhesion [@doi:10.3390/antiox6030049].
Intravenous vitamin C also reduced D-dimer levels in a case study of 17 COVID-19 patients [@doi:10.1080/14787210.2020.1794819].

> D-dimer levels are an important indicator of thrombus formation and breakdown, which are notably elevated in COVID-19 patients [@doi:10.1186/s13054-020-03060-9; @doi:10/ggvd74]. 

Reviewer: ```suggestion
D-dimer levels are an important indicator of thrombus formation and breakdown and are notably elevated in COVID-19 patients [@doi:10.1186/s13054-020-03060-9; @doi:10/ggvd74]. 
```
I think otherwise it sounds like this is saying that thrombus formation & breakdown are noticably elevated

A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than those of SARS-CoV-2 negative patients (24.6 ng/ml; _p_ = 0.004), an association that held when stratifying patients greater than 70 years old [@doi:10.3390/nu12051359].
These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and CT scans of the lungs were obtained upon hospitalization [@doi:10.1101/2020.05.01.20079376].

> A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched diseased controls.

Reviewer: ```suggestion
A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched hospitalized controls.
```
I think this might be more clear than the phrase they use? Just an idea to swap it.

Author: The weird thing from the paper is that they never specify who these people are or what disease they have... at any stage. We can only assume they mean hospitalized

> It is not clear from the study what diseases the control subjects had at the time of their 25-hydroxyvitamin D measurement, which makes it difficult to assess the observations reported.

Reviewer: ```suggestion
It is not clear from the study whether the control subjects had other diseases at the time of their 25-hydroxyvitamin D measurement, which makes it difficult to assess the observations reported.
```
The phrasing seemed a bit unusual to me, would something like this work?

Although this study received significant criticism from the National Institute for Health and Care Excellence (NICE) in the UK [@url:https://www.nice.org.uk/advice/es28/chapter/Key-messages], an independent follow-up statistical analysis supported the findings of the study with respect to the results of cholecalciferol treatment [@doi:10.1101/2020.11.08.20222638].
Another trial of 986 patients hospitalized for COVID-19 in three UK hospitals administered cholecalciferol (≥ 280,000 IU in a time period of 7 weeks) to 151 patients and found an association with a reduced risk of COVID-19 mortality, regardless of baseline 25-hydroxyvitamin D levels [@doi:10.3390/nu12123799].
However, a double-blind, randomized, placebo-controlled trial of 240 hospitalized COVID-19 patients in São Paulo, Brazil administered a single 200,000 IU oral dose of vitamin D.

> At the end of the study, there was a 24 ng/mL difference of 25-hydroxyvitamin D levels in the treatment group versus the placebo group (_p_ = 0.001), 87% of the treatment group were vitamin D sufficieny versus ~11% in the placebo group and supplementation was well tolerated. 

Reviewer 2: Is "sufficieny" the intended word here?  Is "sufficient" more natural?

> At the end of the study, there was a 24 ng/mL difference of 25-hydroxyvitamin D levels in the treatment group versus the placebo group (_p_ = 0.001), 87% of the treatment group were vitamin D sufficieny versus ~11% in the placebo group and supplementation was well tolerated. 

Reviewer 3: ```suggestion
At the end of the study, there was a 24 ng/mL difference of 25-hydroxyvitamin D levels in the treatment group versus the placebo group (_p_ = 0.001), 87% of the treatment group were vitamin D sufficient versus ~11% in the placebo group and supplementation was well tolerated. 
```

> At the end of the study, there was a 24 ng/mL difference of 25-hydroxyvitamin D levels in the treatment group versus the placebo group (_p_ = 0.001), 87% of the treatment group were vitamin D sufficient versus ~11% in the placebo group and supplementation was well tolerated. 

Reviewer: ```suggestion
At the end of the study, there was a 24 ng/mL difference of 25-hydroxyvitamin D levels in the treatment group versus the placebo group (_p_ = 0.001), and 87% of the treatment group were vitamin D sufficient versus ~11% in the placebo group.
Supplementation was well tolerated. 
```

> However, there was no reduction in the length of hospital stay or mortality and no change to any other relevant secondary outcomes were reported [@doi:10.1101/2020.11.16.20232397].

Reviewer: ```suggestion
However, there was no reduction in the length of hospital stay or mortality, and no change to any other relevant secondary outcomes were reported [@doi:10.1101/2020.11.16.20232397].
```

In this review, we report the findings to date of analyses of several dietary supplements and nutraceuticals.
While existing evidence suggests potential benefits of n-3 PUFA and probiotic supplementation for COVID-19 treatment and prophylaxis, clinical data is still lacking, although trials are underway.
Both zinc and vitamin C supplementation in hospitalized patients seem to be associated with positive outcomes; however, further clinical trials are required.

> In any case, vitamin C and zinc intake are part of a healthy diet and likely present minimal risk when supplemented, though their potential prophylactic or therapeutic effects against COVID-19 is yet to be determined.

Reviewer: ```suggestion
In any case, vitamin C and zinc intake are part of a healthy diet and likely present minimal risk when supplemented, though their potential prophylactic or therapeutic effects against COVID-19 are yet to be determined.
```

Author: This was the reviewer suggestion to change are -> is ... I changed one because I wasn't sure myself... I thought using are was fine

Reviewer: I actually think I understand what they're saying now -- "or" makes it weird. Probably the most-correct would be either "any potential prophylactic..." or alternatively "their potential prophylactic and therapeutic effects..." and then use "are", but I doubt it's worth getting too worried about!

On the other hand, mounting evidence from observational studies indicates that there is an association between vitamin D deficiency and COVID-19 incidence has also been supported by meta-analysis [@doi:10.1080/10408398.2020.1841090].
Indeed, scientists are working to confirm these findings and to determine whether a patient's serum 25-hydroxyvitamin D levels are also associated with COVID-19 severity.
Clinical trials are required to determine whether preemptive vitamin D supplementation may mitigate against severe COVID-19.

> In terms of the therapeutic potential of vitamin D, initial evidence from clinical trials is conflicting, but seems to indicate that vitamin D supplementation may reduce COVID-19 severity [@doi:10.1016/j.jsbmb.2020.105751].

Reviewer: ```suggestion
In terms of the therapeutic potential of vitamin D, initial evidence from clinical trials is conflicting but seems to indicate that vitamin D supplementation may reduce COVID-19 severity [@doi:10.1016/j.jsbmb.2020.105751].
```
